docs/7.x/changelog.md

```diff
@@ -1529,6 +1529,11 @@ to learn how to gain insight into how the cluster status changes over time.
 
 ## 5.5 Releases
 
+### 5.5.58 LTS (February 4, 2021)
+#### Improvements
+* Include host record for google metadata server ([#2388](https://github.com/gravitational/gravity/pull/2408), [planet#820](https://github.com/gravitational/planet/pull/820)).
```

Comment:
The numbers don't line up for `[#2388](https://github.com/gravitational/gravity/pull/2408)`